docs/userguide.rst

```diff
@@ -251,13 +253,19 @@ The above example configures 'somename.eth' to resolve to the address of your pr
 Transferring a name
 -------------------
 
-You can transfer ownership of a name you own in the ENS registry to someone else using `setOwner`:
+You can transfer ownership of a name you own in the ENS registry to another trusted account using `setOwner`:
```

Comment:
Basically, I wanted to remove the suggestion that this could be used to transfer to _someone else_.